server/public/assets/scripts/profile.html

```diff
@@ -1,23 +1,49 @@
-<div>
-    I'm a primarily front-end developer with solid JavaScript/jQuery skills, in addition to experience with full MEAN-stack development, as well as Python/Django. However, my familiarity with the back-end has made it easy for me to work closely with back-end developers in my most recent role. I excel at self-directed learning and project work, with a strong record of picking up new technologies quickly and great organizational skills. I'm also very effective working in paired and group programming settings, and I love engaging with other coders in the community through meetups, presentations, hack events, and collaborative learning (I'm the summer cohort leader for PyLadies).
```

Comment:
Don't lose this language: 
I excel at self-directed learning and project work, with a strong record of picking up new technologies quickly and great organizational skills. I'm also very effective working in paired and group programming settings, and I love engaging with other coders in the community through meetups, presentations, hack events, and collaborative learning